src/commands/synthetics/utils.ts

```diff
@@ -802,14 +800,63 @@ export const renderResults = ({
       summary.failedNonBlocking++
     } else {
       summary.failed++
-      hasSucceeded = false
```

Comment:
This `else` translates to `if (resultOutcome === ResultOutcome.Failed)`.

This "hasSucceeded = false" was moved to `handleExit()` [below](https://github.com/DataDog/datadog-ci/pull/822/files#diff-741a985551714cec0c6f7a6423d704525619e8f57db6c4f7c685d95d88a8d0cfR825-R828).

Reply:
Is this comment still valid? I don't see the `hasFailedTests` in the diff.